src/fs.rs

```diff
@@ -103,9 +93,9 @@ impl AccessFs {
     // Roughly read (i.e. not all FS actions are handled).
     /// Gets the access rights identified as read-only according to a specific ABI.
     /// Exclusive with [`from_write()`](AccessFs::from_write).
-    pub fn from_read(abi: ABI) -> BitFlags<Self> {
+    pub fn from_read(abi: ABI) -> Self {
         match abi {
-            ABI::Unsupported => BitFlags::EMPTY,
+            ABI::Unsupported => AccessFs::EMPTY,
```

Comment:
We could implement `AccessFs::none()` instead, but to ease transition it would be nice to still also have the `BitFlags::EMPTY` marked as deprecated.

Reply:
It would be nice to be able to avoid this issue with `AccessNet`, but I don't know how: https://landlock.io/rust-landlock/landlock/trait.Access.html#warning